operator-framework-core/src/main/java/io/javaoperatorsdk/operator/processing/event/DefaultEventSourceManager.java

```diff
@@ -92,7 +95,10 @@ public final void registerEventSource(String name, EventSource eventSource)
       if (e instanceof KubernetesClientException) {
         KubernetesClientException ke = (KubernetesClientException) e;
         if (404 == ke.getCode()) {
```

Comment:
I mean this logic. This is quite specific to CustomResourceEventSource. Cannot be moved there?